Author: Stefan Reiff <kroko87@hotmail.com>
Reason: The macro 'PTHREAD_MUTEX_ERRORCHECK_NP' isn't available in musl libc.
Upstream: No

--- a/PLUGINS/src/osd2web/lib/common.c
+++ b/PLUGINS/src/osd2web/lib/common.c
@@ -1745,7 +1745,7 @@
   locked = 0;
   pthread_mutexattr_t attr;
   pthread_mutexattr_init(&attr);
-  pthread_mutexattr_settype(&attr, PTHREAD_MUTEX_ERRORCHECK_NP);
+  pthread_mutexattr_settype(&attr, PTHREAD_MUTEX_ERRORCHECK);
   pthread_mutex_init(&mutex, &attr);
 }
